src/components/views/Game.js

```diff
@@ -308,7 +316,12 @@ export default function Game() {
             history.push(IncomingMessage.url);
         }
     }, [history]);
-
+    const startTimer = () => {
+        console.log('Send Timer Message!');
+        timerWebSocket.current.send(
+            JSON.stringify({timer: 0})
```

Comment:
What are you sending here? The backend does nothing with this message, no? Why not send en empty string.